<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontPage</class>
 <widget class="QMainWindow" name="FrontPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="Tasks_listWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>421</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>183</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Options_verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="AddTask_pushButton">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>+ Add Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DueToday_pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Due Today</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Urgent_pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Urgent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Calender_pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Calender</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreateAnalytics_pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Create Analytics</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Search_horizontalLayout">
     <item>
      <widget class="QPushButton" name="Search_pushButton">
       <property name="styleSheet">
        <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Search_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="TaskDetails_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="TName_horizontalLayout">
      <item>
       <widget class="QLabel" name="TName_label">
        <property name="text">
         <string>Task name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TName_lineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Priority_horizontalLayout">
      <item>
       <widget class="QComboBox" name="Priority_comboBox">
        <property name="placeholderText">
         <string>Priority</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Desc_horizontalLayout">
      <item>
       <widget class="QLabel" name="Desc_label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="Desc_textEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="DDate_horizontalLayout">
      <item>
       <widget class="QLabel" name="DDate_label">
        <property name="text">
         <string>Due date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="DDate_dateTimeEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>250</y>
       <width>311</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="Extra_horizontalLayout">
      <item>
       <widget class="QComboBox" name="Repeat_comboBox">
        <property name="placeholderText">
         <string>Repeat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="AddTag_comboBox">
        <property name="placeholderText">
         <string>Add Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Reminder_comboBox">
        <property name="placeholderText">
         <string>Reminder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="CreateTask_pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Task</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ViewProfile_pushButton">
       <property name="text">
        <string>View Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Logout_pushButton">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
